docs/PORTABLE_SERVICES.md

```diff
@@ -19,19 +19,18 @@ The primary tool for interacting with Portable Services is `portablectl`,
 and they are managed by the `systemd-portabled` service.
 
 Portable services don't bring anything inherently new to the table. All they do
-is put together known concepts in a slightly nicer way to cover a specific set
-of use-cases in a nicer way.
+is put together known concepts to cover a specific set of use-cases in a
+sligtly nicer way.
 
 ## So, what *is* a "Portable Service"?
 
-A portable service is ultimately just an OS tree, either inside of a directory
-tree, or inside a raw disk image (or a set of images that get layered, see
-[Layered Images](#layered-images)) containing a Linux file system. This tree is called the
```

Comment:
Any reason to remove the layered ref? It is a pretty common use case for container images to be built that way, so it's something people do look for

Reply:
The text tries to be "approachable", but layering is complicated, and when I was reading this, breaks the flow for me. I think it's better to talk about layering later in a separate section. Or in other words, let's keep this part generic and high-level, and talk about how the image is assembled later on.

Reply:
Most readers lose attention after the first couple of paragraphs :-) What about another sentence at the end of the paragraph, rather than in brackets in the middle? That should be better for reading flow?

Reply:
I made it a separate paragraph now: it doesn't seem to fit anywhere. But I fixed the hyperlink as a bonus ;)

Reply:
looks great, thanks!